content/blog/docker-essentials-for-front-end-developers/index.md

```diff
@@ -1,96 +1,99 @@
 ---
 title: Essential Docker for Front End Developers
-date: "2020-03-01"
+date: "2020-03-10"
 author: franleplant
-description:
-  "In this post we are going to cover the steps we took to create the nosleepjavascript.com blog
-  in simple, easy, steps using the latest front end technologies: Gatsby.js (JAMstack), NodeJs, React,
-  GraphQL, Javascript, GitHub Pages and GitHub Actions"
+description: "Docker has become a widely used
+  technology and chances are you are going to have to deal with it eventually, at least superficially, in your Front End
+  career. Lets cover the basic concepts and day to day useful commands you will likely use when dealing with Docker."
```

Comment:
Let's*

Reply:
I never know when to use `lets` or `let's` and even if all of them are valid.